bogota-abril-2019/css-magazine-cover/laura_bruges-task2-magazine/styles/magazine.css

```diff
@@ -0,0 +1,167 @@
+#margin {
+    position: absolute;
+    height: 95vh;
+    width: 32vw;
+    background-color: #FFD102;
+}
+
+h1, h2, h3 {
+    margin: 0
+}
+
+.container {    
+    margin: 10% 6.5%;
+    height: 87%;
+    width: 87%;
+    background-repeat: no-repeat; 
+    background-size: cover;
+}
+
+#girls-container {
+    position: absolute;
+    background-image: url("background_girls.png");
+    z-index: 2;
+}
+
+
+#main-container {
+    position: relative;
+    background-image: url("background.png");
+    color: white;
+    text-align: center;
+    text-transform: uppercase;
+}
+
+#headlines {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 0 2em;

```

Comment:
Across this file im seeing a mixture between units, `vh`, `%` `em`'s can i know how are you telling where to use each one ?

Reply:
Percentages for units related to parent elements, vh and vw for units related to viewport ("absolute" units) and em for units related to the font size (in case of the padding and margins, I was not sure which ones were better, I'll check those out).

Reply:
Padding units are now all percentages!